repository/meteorRepository.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { config } from "../config.js";
+
+const getMeteors = async (date) => {
+  const { monday, friday } = await getMondayAndFridayOfWeek(date);
+
+  const url = `${config.nasaApiUrl}?start_date=${monday}&end_date=${friday}&api_key=${config.nasaApiKey}`;
+
+  const result = await axios.get(url);
+  return result.data.near_earth_objects;
+};
+
+const getMondayAndFridayOfWeek = async (date) => {
+  const givenDate = new Date(date);
+  const dayOfWeek = givenDate.getDay();
+
+  const distanceToMonday = (dayOfWeek + 6) % 7;
+  const monday = new Date(givenDate);
+  monday.setDate(givenDate.getDate() - distanceToMonday);
```

Comment:
too complicate, getDay() returns a number from 1 to 7 (mon - 1, sun - 7) it meens we can use something like this `const distanceToMonday = dayOfWeek - 1`

Reply:
updated